Mathlib/Algebra/Group/Prod.lean

```diff
@@ -12,15 +12,18 @@ import Mathlib.Algebra.GroupWithZero.Units.Basic
 /-!
 # Monoid, group etc structures on `M × N`
 
-In this file we define one-binop (`Monoid`, `Group` etc) structures on `M × N`. We also prove
-trivial `simp` lemmas, and define the following operations on `MonoidHom`s:
+In this file we define one-binop (`Monoid`, `Group` etc) structures on `M × N`.
+We also prove trivial `simp` lemmas, and define the following operations on `MonoidHom`s:
 
 * `fst M N : M × N →* M`, `snd M N : M × N →* N`: projections `Prod.fst` and `Prod.snd`
   as `MonoidHom`s;
 * `inl M N : M →* M × N`, `inr M N : N →* M × N`: inclusions of first/second monoid
   into the product;
 * `f.prod g` : `M →* N × P`: sends `x` to `(f x, g x)`;
-* `f.coprod g : M × N →* P`: sends `(x, y)` to `f x * g y`;
+* (when `P` is commutative) `f.coprod g : M × N →* P` sends `(x, y)` to `f x * g y`;
+* `f.noncommCoprod g comm` is the analogous `M × N →* P`,
```

Comment:
This is in a list of things defined in this file, but is not defined in this file!